src/components/expanded-state/UniqueTokenExpandedState.tsx

```diff
@@ -301,12 +299,10 @@ const UniqueTokenExpandedState = ({
     [showcaseTokens, uniqueId]
   );
 
-  const isSVG = isSupportedUriExtension(asset.image_url || '', ['.svg']);
-  const newImageUrl = svgToPngIfNeeded(asset.image_url, false);
-  const image = isENS && !isSVG ? `${asset.image_url}=s1` : newImageUrl;
```

Comment:
@jxom pointing out i removed this based on https://github.com/rainbow-me/rainbow/pull/3188 LMK if I should revert it